module/VuFind/src/VuFindTest/Integration/MinkTestCase.php

```diff
@@ -741,38 +741,45 @@ protected function waitForPageLoad(
         $session = $this->getMinkSession();
         // Wait for page load to complete:
         $session->wait($timeout, "document.readyState === 'complete'");
-        // Wait for any AJAX requests to complete:
-        $session->wait(
-            $timeout,
-            "typeof $ !== 'undefined' && $.active === 0"
-        );
+        // Check for jQuery (if not loaded, some checks will be skipped):
+        $jQuery = $session->wait($timeout, "typeof $ !== 'undefined'");
+        if ($jQuery) {
+            // Wait for any AJAX requests to complete:
+            $session->wait(
+                $timeout,
+                "typeof $ !== 'undefined' && $.active === 0"
```

Comment:
Can you remove the `typeof` check here since you already did it above?

Reply:
Done.